test/run/ok/float-fmt.wasm-run.ok

```diff
@@ -7,26 +7,18 @@ fixed Float
 3.14159265358979312
 3.14159265358979311600
 exponential Float
-3e+00
-3.1e+00
-3.1416e+00
-3.141592654e+00
-3.1415926535897931e+00
-3.14159265358979312e+00
-3.14159265358979311600e+00
+3e0
```

Comment:
Should we care about this difference?

Reply:
I am not sure. As long as we do not have a specified output float print format, we could maybe accept the difference. At least, it would need to be mentioned in the release memo. I imagine it could affect some systems, in particular tests that compare expected and actual output...